app/views/content_items/_print_meta_data.html.erb

```diff
@@ -0,0 +1,33 @@
+<br/>
+<br/>
+<br/>
+<p>
+  <%= image_tag "open-government-licence.png", class: "print-meta-data-licence" %>
+</p>
+<br/>
+<br/>
+<br/>
+<p>
+  © Crown copyright <%= @content_item.copyright_year %>
+</p>
+<p>
+  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gsi.gov.uk">psi@nationalarchives.gsi.gov.uk</a>.
+</p>
+<p>
+  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
+</p>
+<p>
+  This publication is available at <a href="www.gov.uk/government/publications">www.gov.uk/government/publications</a>
+</p>
+<% unless @content_item.print_meta_data_contact_address.blank? %>
```

Comment:
It would be, yes. But in this case I'm also testing for "" in which case .blank? is less wordy. I can change it, but we don't want to reserve space for an address in the document if the address is "".